app/webhost/_scripts.sh

```diff
@@ -1,12 +1,22 @@
-juno-local
+# EXECUTE THIS SCRIPT FROM THIS DIRECTORY
 
-junod tx wasm store /home/reece/Desktop/cw-webhost/artifacts/cw_webhost.wasm --from=juno1 --keyring-backend=test --chain-id=local-1 --gas=auto --gas-adjustment=1.5 --gas-prices=0.025ujuno --yes
+junod tx wasm store cw_webhost.wasm --from=juno1 --keyring-backend=test --chain-id=local-1 --gas=auto --gas-adjustment=1.5 --gas-prices=0.025ujuno --yes --home=/home/joel/.juno1
 
-junod tx wasm instantiate 1 '{}' --from=juno1 --keyring-backend=test --chain-id=local-1 --label=cw_webhost --gas=auto --gas-adjustment=1.5 --gas-prices=0.025ujuno --yes --no-admin
+sleep 3
 
+junod tx wasm instantiate 1 '{}' --from=juno1 --keyring-backend=test --chain-id=local-1 --label=cw_webhost --gas=auto --gas-adjustment=1.5 --gas-prices=0.025ujuno --yes --no-admin --home=/home/joel/.juno1
 ADDRESS=juno14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9skjuwg8
 
+sleep 3
 
-junod tx wasm execute $ADDRESS '{"new_website":{"name":"test","source":"<html><script>alert(\"popup\")</script><style>body {background-color: lightblue;}</style><h1>Test Website Header</h1><p>Paragraph</p></html>"}}' --from=juno1 --keyring-backend=test --chain-id=local-1 --gas=auto --gas-adjustment=1.5 --gas-prices=0.025ujuno --yes
+# Encode zip file to base64, generate new_web_cmd.txt
+# ENSURE YOU HAVE A FILE NAMED 'web.zip' in the same directory as this script
+python3 encode_zip_to_base64.py
+JSON="$(cat new_web_cmd.txt)"
+junod tx wasm execute $ADDRESS $JSON --from=juno1 --keyring-backend=test --chain-id=local-1 --gas=auto --gas-adjustment=1.5 --gas-prices=0.025ujuno --yes  --home=/home/joel/.juno1
 
-junod q wasm contract-state smart $ADDRESS '{"get_website":{"name":"test"}}'
\ No newline at end of file
+sleep 3
+
+junod q wasm contract-state smart $ADDRESS '{"get_website":{"name":"test"}}' --home=/home/joel/.juno1
```

Comment:
is home required here for the query? if so

```suggestion
junod q wasm contract-state smart $ADDRESS '{"get_website":{"name":"test"}}' --home=$HOME/.juno1
```